### DISCUSSION TITLE
Odd readings, one phase has low voltage (~50 V)

### DISCUSSION THREAD

**Discussion by LordMike** [UtilitechAS/amsreader-firmware#371] [Show and tell]
Hi,

I'm in touch with Radius (Danish utility grid) about this - but I wanted to show off here, as I don't think this happens often. I saw these values running a previous version of the ESP code to read the serial output, but an Amsleser is way cooler - so I got that.

![image](https://user-images.githubusercontent.com/1027111/205956071-b10f5962-aef5-494e-b218-8f8780a6b522.png)

One of my phases, in my past setup I had it as L2, but here it's L1-L3 (which is L2?), is way below the 220 V target. The corresponding ampitude on the right is higher, possibly as a consequence of this.. Which also means I have some appliance running on 50V, but I don't know what.

I do have weird stuff in the house, like a tumble dryer that refuses to run on some wall outlets, or lights that flicker - I'm not sure if it's related at all - but I'm tracking down issues one by one :).

Radius said something along the lines of "Ehm, that's really odd... we'll get back to you". So yay me :)

**Comment by ArnieO** [comment]
1. **Call an electrician - please!!**
2. Voltage is measured between phases, and L1-L3 means that it is the voltage measured between lines 1 and 3. Current is measured in each phase wire L1, L2, L3. But I agree - some of your outlets should show 50 VAC if you mesure. And for sure: applicances will not work.

**Reply by LordMike** [reply]
I have yet to find an outlet that reads 50V (part of me suspects the meter may be broken).

As for the electrician, Radius wanted to send a technician, but also wanted to get back to me - it sounded like this hasn't happened before or is very rare. The other electricians I've had out and about have never found anything that would make them mention the 50V, and I kinda suspect that with all the work I've had done, they'd have found a 50V line at some point.. But who knows.

Much like an active volcano, this seems to have been an issue for years now, and I'm still here.. Buut I'm going to nag Radius a bit more. Maybe also pull fuses and see when the amps drop, like Egil suggested. :)

**Reply by ArnieO** [reply]
What made me suspect a problem in your installation is that you mentioned _"...weird stuff in the house, like a tumble dryer that refuses to run on some wall outlets, or lights that flicker..."_

To me it sounds related - but an electrician (or Radius technician) should be able to find the issue.
▶️ Measure voltages at the connection point of the meter. If voltages between all 3 phases are correct, then the meter is faulty.

**Reply by LordMike** [reply]
Sooeh.. I wrote back to Radius, they called me .. and now I'm getting a visit within a few hours. Will update.

**Reply by LordMike** [reply]
Technician was here. Apparently I only have two phases in the building, so the L2 one isn’t physically connected. This can explain the low voltage, but does it explain this usage (was cooking).

If I multiply the amps with the respective voltages, I get 3.3kW for L1+L3 which matches the gauge in the top, and 1.9kW for L2, which then isn’t included there. So maybe it’s not wrong ?

![B82363FD-145B-4BA0-93F4-AA744EC69DB9](https://user-images.githubusercontent.com/1027111/205994932-868fcce6-eaa9-4182-9389-e25678165bcd.png)

**Reply by ArnieO** [reply]
Interesting case. 

Does that mean they have hooked up a 3-phase meter in a two-phase installation? Sounds weird.
What model number does the meter report (see Config/Meter)?

Did the technician explain how you can be hooked up to only two phases when the meter reports current on 3 phases?
And that current on L2 is very high... what is the size of your main fuse?

**Reply by gskjold** [reply]
Interesting case. It could be that the meter reports a voltage, but not a current. If L2 current is missing when a voltage exists for all three phases, L2 current is automatically calculated. Would need to see debug output to determine if this is the case.

**Reply by LordMike** [reply]
The meter is a three phase, part of why I've assumed I had three phases.. As I got it described last night, there are three ~phases~ wires hooked up to the meter, but on the other end, at the road side, there is a plug that only connects two phases... so maybe the cabling is just ready for three phases if I ever upgraded to that.

I'm told that "at the front" I have 25A. This apparently means that there is a fuse out in the electrical junction box roadside, that provides 2x 25A for 50A total. Inside I have an array of fuses for various groups, typically 10A or 16A. Interestingly, one fuse for my oven/stove is three phases (also part of why I assumed three) - but even if all three are hooked up internally, it will only be provided power on two. 

What's weird is the electrician showed me the wires from the meter, in which three wires are hooked up roadside, but only two move on internally. So it is "physically impossible" for the meter to measure anything. So the measurement I'm seeing _must_ be a fluke of some sorts. The current falls and rises given my other usage, but that must be induced somehow?.. 

... or maybe the reading of current is "fake" and calculated based on other input?

As I showed in the screenshot, the wattage reported doesn't match the `voltage * amps` for all phases, which indicates that the meter _knows_ it's not using anything on L2 even if it reports voltage and current for it.

**Reply by LordMike** [reply]
Model number for meter is blank in AMS. I read it physically as an `OMNIPOWER - THREE-PHASE METER`. Will attach photo.

Also. I put in 12 kWp for solar generation, but it's not up yet. So actual production is 0. Which will also be fun, because if I actually produced 12 kW and wanted to export 100%, I'd be limited to 11kW given 50A/220V... So yay :).. It's probably fine though.

![image](https://user-images.githubusercontent.com/1027111/206134164-f42bee59-0334-449e-b9aa-978d5b887317.png)

![8DE47E5B-E60A-4F82-8903-EAD37830EB7F](https://user-images.githubusercontent.com/1027111/206134560-d0f39b3c-0d59-4d71-933d-5f6439b484a9.jpeg)

**Reply by LordMike** [reply]
> Interesting case. It could be that the meter reports a voltage, but not a current. If L2 current is missing when a voltage exists for all three phases, L2 current is automatically calculated. Would need to see debug output to determine if this is the case.

Ooh, this may actually be the case. In my previous setup, L2 Current was always reported as 0A, which I found very odd (assuming I had three phases). I thought it may have bounced, but I just checked some history in HASS, and it was 0A all december.

I've enabled telnet debugging and connected in, and sure enough a frame popped out. Provided below.

<details>
<summary>Kamstrup OMNIPower frame</summary>

```
(V t:76946936ms) (unwrapData)(C0) HDLC frame:
(V t:76946941ms) 7E A1 E9 41  03 13 C6 37  E6 E7 00 DB  08 4B 41 4D
(V t:76946942ms) 45 01 AC 6F  09 82 01 D0  30 00 35 3F  7C 3E 22 E2
(V t:76946942ms) D3 96 B3 1C  7A 17 5C FA  BB 44 30 04  36 E7 2E 29
(V t:76946943ms) FB 18 E4 9E  9D AC C5 26  D8 33 3B 1F  A1 51 84 E9
(V t:76946943ms) 13 C6 AD FB  3C 82 1C 56  C3 F2 37 B5  A2 96 9D 55
(V t:76946944ms) 28 E1 A9 56  50 D5 61 CE  D6 BC 22 4E  DF 94 6E DC
(V t:76946944ms) B4 AB 81 B1  7E 64 B9 3E  CE FF 47 2B  43 BC 95 73
(V t:76946945ms) 61 27 AB 55  C0 9F A7 B7  7F 48 CC 88  59 4C C0 A9
(V t:76946945ms) 43 6A 4A E5  71 46 ED 2B  D9 F5 97 A4  9F B8 E8 FF
(V t:76946945ms) 6A 67 E6 BE  7A 2F 14 8A  5C EC F8 FC  AA 45 D2 41
(V t:76946946ms) 6A B5 87 D0  B1 D5 4B 04  2C A3 01 CC  A5 3B 02 F8
(V t:76946946ms) D5 A8 97 22  8C 20 7A 6A  C2 44 37 AC  67 BA DC 93
(V t:76946947ms) CB 6A EA B1  5F AA 8F 35  70 0D DA AF  86 F9 56 72
(V t:76946947ms) 08 41 69 4B  D3 0B D5 B6  1B 2C 34 35  40 DD 4F F5
(V t:76946948ms) D5 F6 01 4F  63 6A 95 ED  DD DD A6 BD  73 D3 19 3A
(V t:76946948ms) C2 8C 70 55  13 05 15 AE  34 1D 4B 21  AA 2C 86 0B
(V t:76946949ms) 8B 7F 1F 10  DB DC 9A D8  E5 5D 32 43  34 C1 D3 7B
(V t:76946949ms) EC 0A 48 97  C5 5E DA AF  9F 67 91 E4  50 AB FB E8
(V t:76946955ms) 94 8B 73 5E  EF 1E F5 5A  B2 5B AD C5  D7 8C 65 5F
(V t:76946956ms) 50 4E 0D 4E  26 5B 60 3A  3A DC 52 E9  B3 A3 1B 10
(V t:76946957ms) 7E 98 CA 9B  27 36 1D 33  B2 96 09 33  CE 41 1A 12
(V t:76946957ms) C5 B1 C0 8F  0E 3A D5 21  21 01 D8 D4  12 27 FA 10
(V t:76946958ms) CF 4C EA 56  6D A2 10 3A  59 D7 97 81  F3 75 EB 2A
(V t:76946958ms) D3 9A D4 E7  22 7B 05 BB  B6 2F E0 53  8F D3 DB D8
(V t:76946959ms) 11 BD 3C C7  E8 43 B4 DF  9E AC 6B A1  BE D1 0F BC
(V t:76946959ms) DF AB FC FB  F8 A3 77 78  60 37 65 6A  02 3B 3A D1
(V t:76946960ms) 7C 4D 34 3E  95 EE DA 0E  3C B7 85 AC  2B A1 91 FC
(V t:76946960ms) C3 89 B5 05  3C 81 A4 C4  68 E6 D7 61  55 3F 49 D7
(V t:76946960ms) 57 9B E2 0C  D6 FB 98 6A  39 85 82 51  66 3C 48 AF
(V t:76946961ms) 88 1B EC 13  67 13 C2 78  C4 53 74 0D  F2 89 8E BC
(V t:76946961ms) CC 32 6D CE  34 9D 0A FD  5A A4 7E
(V t:76946962ms) (unwrapData)(C0) LLC frame:
(V t:76946962ms) E6 E7 00 DB  08 4B 41 4D  45 01 AC 6F  09 82 01 D0
(V t:76946966ms) 30 00 35 3F  7C 3E 22 E2  D3 96 B3 1C  7A 17 5C FA
(V t:76946967ms) BB 44 30 04  36 E7 2E 29  FB 18 E4 9E  9D AC C5 26
(V t:76946967ms) D8 33 3B 1F  A1 51 84 E9  13 C6 AD FB  3C 82 1C 56
(V t:76946968ms) C3 F2 37 B5  A2 96 9D 55  28 E1 A9 56  50 D5 61 CE
(V t:76946968ms) D6 BC 22 4E  DF 94 6E DC  B4 AB 81 B1  7E 64 B9 3E
(V t:76946969ms) CE FF 47 2B  43 BC 95 73  61 27 AB 55  C0 9F A7 B7
(V t:76946969ms) 7F 48 CC 88  59 4C C0 A9  43 6A 4A E5  71 46 ED 2B
(V t:76946970ms) D9 F5 97 A4  9F B8 E8 FF  6A 67 E6 BE  7A 2F 14 8A
(V t:76946970ms) 5C EC F8 FC  AA 45 D2 41  6A B5 87 D0  B1 D5 4B 04
(V t:76946971ms) 2C A3 01 CC  A5 3B 02 F8  D5 A8 97 22  8C 20 7A 6A
(V t:76946971ms) C2 44 37 AC  67 BA DC 93  CB 6A EA B1  5F AA 8F 35
(V t:76946972ms) 70 0D DA AF  86 F9 56 72  08 41 69 4B  D3 0B D5 B6
(V t:76946972ms) 1B 2C 34 35  40 DD 4F F5  D5 F6 01 4F  63 6A 95 ED
(V t:76946973ms) DD DD A6 BD  73 D3 19 3A  C2 8C 70 55  13 05 15 AE
(V t:76946973ms) 34 1D 4B 21  AA 2C 86 0B  8B 7F 1F 10  DB DC 9A D8
(V t:76946974ms) E5 5D 32 43  34 C1 D3 7B  EC 0A 48 97  C5 5E DA AF
(V t:76946974ms) 9F 67 91 E4  50 AB FB E8  94 8B 73 5E  EF 1E F5 5A
(V t:76946975ms) B2 5B AD C5  D7 8C 65 5F  50 4E 0D 4E  26 5B 60 3A
(V t:76946975ms) 3A DC 52 E9  B3 A3 1B 10  7E 98 CA 9B  27 36 1D 33
(V t:76946979ms) B2 96 09 33  CE 41 1A 12  C5 B1 C0 8F  0E 3A D5 21
(V t:76946982ms) 21 01 D8 D4  12 27 FA 10  CF 4C EA 56  6D A2 10 3A
(V t:76946982ms) 59 D7 97 81  F3 75 EB 2A  D3 9A D4 E7  22 7B 05 BB
(V t:76946983ms) B6 2F E0 53  8F D3 DB D8  11 BD 3C C7  E8 43 B4 DF
(V t:76946983ms) 9E AC 6B A1  BE D1 0F BC  DF AB FC FB  F8 A3 77 78
(V t:76946984ms) 60 37 65 6A  02 3B 3A D1  7C 4D 34 3E  95 EE DA 0E
(V t:76946984ms) 3C B7 85 AC  2B A1 91 FC  C3 89 B5 05  3C 81 A4 C4
(V t:76946985ms) 68 E6 D7 61  55 3F 49 D7  57 9B E2 0C  D6 FB 98 6A
(V t:76946985ms) 39 85 82 51  66 3C 48 AF  88 1B EC 13  67 13 C2 78
(V t:76946986ms) C4 53 74 0D  F2 89 8E BC  CC 32 6D CE  34 9D 0A FD
(V t:76946986ms)
(V t:76946987ms) (unwrapData)(C0) GCM frame:
(V t:76946987ms) DB 08 4B 41  4D 45 01 AC  6F 09 82 01  D0 30 00 35
(V t:76946988ms) 3F 7C 0F 00  00 00 00 0C  07 E6 0C 07  03 0A 06 14
(V t:76946989ms) FF 80 00 00  02 41 0A 0E  4B 61 6D 73  74 72 75 70
(V t:76946993ms) 5F 56 30 30  30 31 09 06  01 01 01 08  00 FF 06 00
(V t:76946993ms) 2D 80 E2 09  06 01 01 02  08 00 FF 06  00 00 00 00
(V t:76946994ms) 09 06 01 01  03 08 00 FF  06 00 00 C7  76 09 06 01
(V t:76946994ms) 01 04 08 00  FF 06 00 0C  39 0B 09 06  01 01 00 00
(V t:76946995ms) 01 FF 06 01  44 B5 E4 09  06 01 01 01  07 00 FF 06
(V t:76946995ms) 00 00 01 CE  09 06 01 01  02 07 00 FF  06 00 00 00
(V t:76946997ms) 00 09 06 01  01 03 07 00  FF 06 00 00  00 00 09 06
(V t:76946998ms) 01 01 04 07  00 FF 06 00  00 01 13 09  06 00 01 01
(V t:76946998ms) 00 00 FF 09  0C 07 E6 0C  07 03 0A 06  14 FF 80 00
(V t:76946999ms) 00 09 06 01  01 20 07 00  FF 12 00 DA  09 06 01 01
(V t:76946999ms) 34 07 00 FF  12 00 3A 09  06 01 01 48  07 00 FF 12
(V t:76947000ms) 00 E0 09 06  01 01 1F 07  00 FF 06 00  00 00 5E 09
(V t:76947000ms) 06 01 01 33  07 00 FF 06  00 00 00 00  09 06 01 01
(V t:76947001ms) 47 07 00 FF  06 00 00 00  BE 09 06 01  01 15 07 00
(V t:76947001ms) FF 06 00 00  00 8E 09 06  01 01 29 07  00 FF 06 00
(V t:76947005ms) 00 00 00 09  06 01 01 3D  07 00 FF 06  00 00 01 40
(V t:76947006ms) 09 06 01 01  21 07 00 FF  12 00 52 09  06 01 01 35
(V t:76947008ms) 07 00 FF 12  00 64 09 06  01 01 49 07  00 FF 12 00
(V t:76947009ms) 57 09 06 01  01 0D 07 00  FF 12 00 55  09 06 01 01
(V t:76947009ms) 16 07 00 FF  06 00 00 00  00 09 06 01  01 2A 07 00
(V t:76947010ms) FF 06 00 00  00 00 09 06  01 01 3E 07  00 FF 06 00
(V t:76947010ms) 00 00 00 09  06 01 01 16  08 00 FF 06  00 00 00 00
(V t:76947011ms) 09 06 01 01  2A 08 00 FF  06 00 00 00  00 09 06 01
(V t:76947011ms) 01 3E 08 00  FF 06 00 00  00 00 09 06  01 01 15 08
(V t:76947012ms) 00 FF 06 00  0A E4 C8 09  06 01 01 29  08 00 FF 06
(V t:76947012ms) 00 00 00 00  09 06 01 01  3D 08 00 FF  06 00 22 9C
(V t:76947013ms) 1A F2 89 8E  BC CC 32 6D  CE 34 9D 0A  FD
(V t:76947014ms) (unwrapData)(C0) DLMS frame:
(V t:76947014ms) 0F 00 00 00  00 0C 07 E6  0C 07 03 0A  06 14 FF 80
(V t:76947014ms) 00 00 02 41  0A 0E 4B 61  6D 73 74 72  75 70 5F 56
(V t:76947018ms) 30 30 30 31  09 06 01 01  01 08 00 FF  06 00 2D 80
(V t:76947019ms) E2 09 06 01  01 02 08 00  FF 06 00 00  00 00 09 06
(V t:76947020ms) 01 01 03 08  00 FF 06 00  00 C7 76 09  06 01 01 04
(V t:76947020ms) 08 00 FF 06  00 0C 39 0B  09 06 01 01  00 00 01 FF
(V t:76947021ms) 06 01 44 B5  E4 09 06 01  01 01 07 00  FF 06 00 00
(V t:76947021ms) 01 CE 09 06  01 01 02 07  00 FF 06 00  00 00 00 09
(V t:76947022ms) 06 01 01 03  07 00 FF 06  00 00 00 00  09 06 01 01
(V t:76947022ms) 04 07 00 FF  06 00 00 01  13 09 06 00  01 01 00 00
(V t:76947023ms) FF 09 0C 07  E6 0C 07 03  0A 06 14 FF  80 00 00 09
(V t:76947023ms) 06 01 01 20  07 00 FF 12  00 DA 09 06  01 01 34 07
(V t:76947024ms) 00 FF 12 00  3A 09 06 01  01 48 07 00  FF 12 00 E0
(V t:76947024ms) 09 06 01 01  1F 07 00 FF  06 00 00 00  5E 09 06 01
(V t:76947025ms) 01 33 07 00  FF 06 00 00  00 00 09 06  01 01 47 07
(V t:76947025ms) 00 FF 06 00  00 00 BE 09  06 01 01 15  07 00 FF 06
(V t:76947026ms) 00 00 00 8E  09 06 01 01  29 07 00 FF  06 00 00 00
(V t:76947026ms) 00 09 06 01  01 3D 07 00  FF 06 00 00  01 40 09 06
(V t:76947027ms) 01 01 21 07  00 FF 12 00  52 09 06 01  01 35 07 00
(V t:76947027ms) FF 12 00 64  09 06 01 01  49 07 00 FF  12 00 57 09
(V t:76947033ms) 06 01 01 0D  07 00 FF 12  00 55 09 06  01 01 16 07
(V t:76947034ms) 00 FF 06 00  00 00 00 09  06 01 01 2A  07 00 FF 06
(V t:76947035ms) 00 00 00 00  09 06 01 01  3E 07 00 FF  06 00 00 00
(V t:76947035ms) 00 09 06 01  01 16 08 00  FF 06 00 00  00 00 09 06
(V t:76947036ms) 01 01 2A 08  00 FF 06 00  00 00 00 09  06 01 01 3E
(V t:76947036ms) 08 00 FF 06  00 00 00 00  09 06 01 01  15 08 00 FF
(V t:76947037ms) 06 00 0A E4  C8 09 06 01  01 29 08 00  FF 06 00 00
(V t:76947037ms) 00 00 09 06  01 01 3D 08  00 FF 06 00  22 9C 1A
(V t:76947038ms) (readHanPort)(C0) Received valid DLMS at 47
(V t:76947039ms) (readHanPort)(C0) Using application data:
(V t:76947039ms) 02 41 0A 0E  4B 61 6D 73  74 72 75 70  5F 56 30 30
(V t:76947039ms) 30 31 09 06  01 01 01 08  00 FF 06 00  2D 80 E2 09
(V t:76947040ms) 06 01 01 02  08 00 FF 06  00 00 00 00  09 06 01 01
(V t:76947040ms) 03 08 00 FF  06 00 00 C7  76 09 06 01  01 04 08 00
(V t:76947044ms) FF 06 00 0C  39 0B 09 06  01 01 00 00  01 FF 06 01
(V t:76947045ms) 44 B5 E4 09  06 01 01 01  07 00 FF 06  00 00 01 CE
(V t:76947046ms) 09 06 01 01  02 07 00 FF  06 00 00 00  00 09 06 01
(V t:76947048ms) 01 03 07 00  FF 06 00 00  00 00 09 06  01 01 04 07
(V t:76947048ms) 00 FF 06 00  00 01 13 09  06 00 01 01  00 00 FF 09
(V t:76947049ms) 0C 07 E6 0C  07 03 0A 06  14 FF 80 00  00 09 06 01
(V t:76947049ms) 01 20 07 00  FF 12 00 DA  09 06 01 01  34 07 00 FF
(V t:76947050ms) 12 00 3A 09  06 01 01 48  07 00 FF 12  00 E0 09 06
(V t:76947050ms) 01 01 1F 07  00 FF 06 00  00 00 5E 09  06 01 01 33
(V t:76947051ms) 07 00 FF 06  00 00 00 00  09 06 01 01  47 07 00 FF
(V t:76947051ms) 06 00 00 00  BE 09 06 01  01 15 07 00  FF 06 00 00
(V t:76947052ms) 00 8E 09 06  01 01 29 07  00 FF 06 00  00 00 00 09
(V t:76947052ms) 06 01 01 3D  07 00 FF 06  00 00 01 40  09 06 01 01
(V t:76947053ms) 21 07 00 FF  12 00 52 09  06 01 01 35  07 00 FF 12
(V t:76947053ms) 00 64 09 06  01 01 49 07  00 FF 12 00  57 09 06 01
(V t:76947060ms) 01 0D 07 00  FF 12 00 55  09 06 01 01  16 07 00 FF
(V t:76947060ms) 06 00 00 00  00 09 06 01  01 2A 07 00  FF 06 00 00
(V t:76947061ms) 00 00 09 06  01 01 3E 07  00 FF 06 00  00 00 00 09
(V t:76947061ms) 06 01 01 16  08 00 FF 06  00 00 00 00  09 06 01 01
(V t:76947062ms) 2A 08 00 FF  06 00 00 00  00 09 06 01  01 3E 08 00
(V t:76947062ms) FF 06 00 00  00 00 09 06  01 01 15 08  00 FF 06 00
(V t:76947063ms) 0A E4 C8 09  06 01 01 29  08 00 FF 06  00 00 00 00
(V t:76947063ms) 09 06 01 01  3D 08 00 FF  06 00 22 9C  1A
(V t:76947347ms) (EnergyAccounting) Adding 0.0013 kWh import
(V t:76947351ms) (EnergyAccounting)  calculating threshold, currently at 0
(V t:76947351ms) (EnergyAccounting)  new threshold 0
```

</details>

**Reply by gskjold** [reply]
Thanks, as suspected OBIS 51.7.0 is 0:
09 06 01 01 **33 07 00** FF 06 **00 00 00 00**

And OBIS 52.7.0 reports 58V:
09 06 01 01 **34 07 00** FF 12 **00 3A** 

I will think about possible adjustments in the code.

As for adding I1*U1 + I3*U3, I am not sure how that works with only two phases, but for a three phase system it would not be valid. I would not be the best to explain this, so I would recommend reading up on it elsewhere, but for reference, when you calculate the maximum wattage of a three phase breaker would be: amperage * system voltage * sqrt(3). Ex a 40A three phase main breaker in a 400V system would give you about 27kw available power.

In the case of your screenshot above, could it be that 230V * ((1.05+1.96)/2) * sqrt(2) is a valid formula? I'm on shaky ground here, but the numbers match

**Reply by ArnieO** [reply]
I am puzzled regarding the voltage indication, where you have voltage on two of the readings. 
This only makes sense if this is a 400V system, meaning that from "the street" you get two power wires plus the null-wire. 
Then there will be 230V from each of two phases towards the null-wire, and what could be just induced noise when measuring between the L2 terminal and null-wire. 
But apparently you have selected 230V in your "Distribution system" setting. 
I think it should be set to 400V in your case.

**Reply by LordMike** [reply]
I had no idea what to put in the distribution system setting, there was no recommendation - to the best of my knowledge I only have 220/230V in all outlets, including high users like the stove. I've selected 400V now, and the L2 current dropped to 0A (screenshot).

Exciting stuff.. :) No clue on calculations.. I'm sticking to entry level electrical-maths :)

![image](https://user-images.githubusercontent.com/1027111/206173125-1bee339a-f7c1-4623-8d3b-040d81df015b.png)

**Reply by ArnieO** [reply]
The installation can be 400V even if all your outlets are 230V. The magic of a 3-phase system... :wink:

The answer is that you must look at the text on your meter. If 400 V is mentioned at all, you have a 400V system. 
In your case: 
![image](https://user-images.githubusercontent.com/10295178/206471098-d4f6df29-b317-4065-a814-f6e1a276c533.png)
